modules/economy-system/configs/config.json

```diff
@@ -196,6 +196,16 @@
       "default": 100,
       "type": "integer",
       "description": "The weekly reward"
+    },
+    {
+      "field_name": "PublicCommandReplies",
```

Comment:
We usually try to keep config field_names starting with a lower letter, I suggest renaming it if possible (the whole config files has lowercase-field-names except this option)